core/spec/models/spree/stock_item_spec.rb

```diff
@@ -200,40 +194,37 @@
       end
     end
 
-    context "binary_inventory_cache is set to true" do
-      let(:binary_inventory_cache) { true }
+    context "inventory_cache_threshold is set" do
+      before do
+        Spree::Config.inventory_cache_threshold = inventory_cache_threshold
+      end
 
-      context "in_stock? changes" do
-        it "touches its variant" do
-          expect do
-            subject.set_count_on_hand(0)
-          end.to change { subject.variant.updated_at }
-        end
+      let(:inventory_cache_threshold) { 5 }
 
-        # Regression spec
-        context "stock changes to below zero" do
-          it "touches its variant" do
-            expect do
-              subject.set_count_on_hand(-1)
-            end.to change { subject.variant.updated_at }
-          end
-        end
+      it "count on hand falls below threshold" do
+        expect do
+          subject.set_count_on_hand(3)
+        end.to change { subject.variant.updated_at }
```

Comment:
Under MySQL, timestamps only have 1 second precision. This has to either be tested differently, or an `update_column` or `update_all` set updated_at to a known past value.